hydrolib/core/io/mdu/models.py

```diff
@@ -492,12 +496,16 @@ class Geometry(INIBasedModel):
     _split_to_list = get_split_string_on_delimiter_validator(
         "frictfile",
         "structurefile",
+        delimiter=";",
+    )
+
+    _split_to_list2 = get_split_string_on_delimiter_validator(
```

Comment:
Do you think it would be possible to use the field delimiter value for parsing as well? So that we do not have to create these validators "manually" again with the correct (=same) delimiter?

Reply:
I'd love to, but did not manage to get that to work. Inside the member variable definitions of a class (here: `Geometry`) we cannot yet access that class'es own class methods, e.g., this is not allowed:
```python
    # THIS DOES NOT WORK:
    _split_to_list = get_split_string_on_delimiter_validator(
        "frictfile",
        "structurefile",
        delimiter=Geometry.get_list_field_delimiter("frictfile"),
    )
```
Unless you know a trick, I'd say let's accept this for now.

Reply:
@arthurvd Might https://github.com/Deltares/HYDROLIB-core/pull/222 work?